src/core_ocean/shared/mpas_ocn_vmix_cvmix.F

```diff
@@ -353,7 +357,16 @@ subroutine ocn_vmix_coefs_cvmix_build(meshPool, statePool, forcingPool, diagnost
 
       call mpas_timer_start('cvmix cell loop', .false.)
       do kpp_stage = 1,2
-      !$omp do schedule(runtime)
+      ! TODO, mdt: determine where the openmp data race is coming from in this loop
+      !!$omp parallel
+      !!$omp do schedule(runtime) &
+      !!$omp private(iCell, invAreaCell, k, RiSmoothed, RiTemp, BVFSmoothed, nsmooth, &
+      !!$omp         langmuirNumber, langmuirEnhancementFactor, bulkRichardsonNumberStop, &
+      !!$omp         bulkRichardsonFlag, topIndex, kIndexOBL, deltaVelocitySquared, sigma, &
+      !!$omp         OBLDepths, interfaceForcings, surfaceAverageIndex, sfc_layer_depth, kav, &
+      !!$omp         i, iEdge, factor, normalVelocitySum, NormalVelocityAv, &
+      !!$omp         delU2, potentialDensitySum, blTemp) &
+      !!$omp firstprivate(cvmix_variables, Nsqr_iface, turbulentScalarVelocityScale)
```

Comment:
This loop has the OpenMP directives commented out since there is a race condition that is yet to be resolved.  This will be resolved in a future PR (the work in this PR need to be merged for future work to build upon).  I'm also putting off resolving this race condition until the other work on CVMIX is merged.